openhtf/output/web_gui/src/app/station.service.ts

```diff
@@ -98,7 +102,9 @@ export class StationService extends SubscriptionService {
   onmessage(msg) {
     this.reachable = true;
     let parsedData = JSON.parse(msg.data);
-    let test_uid = parsedData.test_uid, state = parsedData.state;
+    let test_uid = parsedData.test_uid;
+    let state = parsedData.state;
+    state.test_uid = test_uid  // Add test_uid to the test state.
```

Comment:
Not a huge fan of stashing this directly on state, because it loses the parity between state here and the state that was sent to us. But I don't have a better suggestion short of adding a layer above both. Just a note, not necessarily a blocker.